aikau/src/main/resources/alfresco/forms/ControlRow.js

```diff
@@ -179,11 +179,38 @@ define(["alfresco/layout/HorizontalWidgets",
        * @param {object} widget The widget to validate
        * @param {number} index The index of the widget to validate
        */
-      validateChildFormControlValue: function alfresco_forms_ControlRow__validateChildFormControlValue(widget, index) {
+      validateChildFormControlValue: function alfresco_forms_ControlRow__validateChildFormControlValue(widget, /*jshint unused:false*/ index) {
          if (typeof widget.validateFormControlValue === "function")
          {
             widget.validateFormControlValue();
          }
+      },
+
+      /**
+       * Iterates over the child form controls and publishes the value of each one.
+       * 
+       * @instance
+       * @param {Deferred} deferred A deferred object can optionally be passed. This will only be resolved as widget value
```

Comment:
Is it worth using square brackets to indicate the parameter is optional? http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values